docs/announcement.md

```diff
@@ -1,140 +1,31 @@
 
-* [Navigate to Flipkart](clients.md#navigate-to-flipkart): To navigate to any flipkart native page
-* [Notify page location change](clients.md#notify-page-location-change-react-native-only): To notify that your page location has changed (React native only)
 
+You must update the Deeplinks that support Ultra application within Flipkart to use the new `clientId` as a parameter.
 
-##New v2 Server Side APIs
-_09/Aug/2018_
+**How to handle `clientId` inconsistency?**
 
-We have updated APIs for following
+When you open the Ultra application with a specific `clientId`, it should carry the same `clientId` parameter value for further navigations to avoid inconsistencies. If `clientId` value varies, it might cause problems in case of payment redirections. The post-payment redirect URL (`fapp://action?value={"screenType":"ultra","params":{"url":"<URLENCODED_URL_TO_OPEN_WITHIN_ULTRA>","clientId":"<YOUR_CLIENT_ID>","postPaymentFlow":true,"success":<TRUE_OR_FALSE>}}`) will show a new `clientId` value which will differ from its first value at the time of opening of the application. An example of this scenario is that if user opened the Ultra application with `clientId` value as `mmt`, after the update, the new value becomes as `mmt.flight`. This may cause issues when the user has a Deeplink of SMS or WhatsApp carrying an older `clientId` value.
 
-* New way for generating [secureToken](backend.md#security). Earlier we used to just take company name as clientId. From now on `clientId = <companyName>.<appId>`
-* V2 for get active offers. [Active offers](backend.md#get-offers-list)
-* V2 for payment APIs
-    * [PaymentToken](backend.md#payment-token).  Added category in request.
-    * [Query](backend.md#query). Linked to new secureToken.
-    * [Refund](backend.md#refund). Linked to new secureToken.
-* Do not forget to update your clientId in post payment [fapp:// redirect](clients.md#start-payment) too.
+This further impacts the navigation stack which will appear from `Flipkart Homepage ➡ Ultra (clientId = mmt)` to `Flipkart Homepage ➡ Ultra (clientId = mmt) ➡ Payments ➡ Ultra (clientId = mmt.flight)`. One can capture this when a user has pressed the back button and retraced the navigation stack but same goes unnoticed once the payment cycle is complete.
 
-## OMS contracts updated
-_12/July/2018_
+Thus, you need to change the `clientId` value from old to new one simultaneously across all the systems that depend on it. Some of the examples where this inconsistency may arise are SMS or WhatsApp, payment redirections or Ultra landing page. Everything should be in synchronization with respect to the `clientId` value in the application.
 
-After feedback from merchants we have decided to change the following in our v2 OMS contracts.
-
-* Addition of orderUrl in [Order](oms.md#order).
-* Addition of brand, product, customerName, quantity in [Item](oms.md#item).
-
-----
-
-## Root Cause Analysis of Order failures on MMT (07/07/2018)
```

Comment:
Where are you moving the remaining ones ?

Reply:
These will be moved to User Notifications section as per the structure created of Ultra document.

Reply:
Okay